src/chart/main/layout.cpp

```diff
@@ -20,6 +20,17 @@ void Layout::setBoundary(const Geom::Rect &boundary,
 	this->boundary = boundary;
 	auto rect = style.contentRect(boundary, em);
 
+	auto footerHeight =
```

Comment:
Similar code 3 times. Maybe extract this to a function?

Reply:
ok

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/349/commits/e241f1a8780415d2ac9a4a946d0138fc947bf860